schemas/panels/stat/stat.cue

```diff
@@ -11,7 +11,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package stat
+package model
```

Comment:
is it really a good thing to change the package name and have the same for everything ?

Reply:
well imho as I put in the description _I don't think this is an issue (it's just another rule to follow to develop plugins)_. It shouldn't cause any kind of conflict since each plugin is evaluated independently.